app/chat.py

```
@@ -649,6 +576,109 @@ def run_test_queries(conn) -> None:
+    
+    if is_postgres:
+        # PostgreSQL retrieval
+        from app.retrieval.postgres import get_relevant_chunks as postgres_retrieval
+        return postgres_retrieval(store, query, max_chunks)
+    else:
+        # ChromaDB retrieval
+        from app.retrieval.chroma import search_chunks
+        return search_chunks(store, query, max_chunks)
+
+
+def answer_query(store, query: str) -> str:
```

Ruff (0.8.2)

Redefinition of unused `answer_query` from line 39

(F811)